TestCases/harmonic_balance/HB.cfg

```diff
@@ -4,45 +4,53 @@
 % Case description: Time spectral calc. of flow around a NACA 64A010 airfoil   %
 % Author: Thomas D. Economon                                                   %
 % Institution: Stanford University                                             %
-% Date: 2011.11.02                                                             %
+% Date: 2016.20.09                                                             %
 % File Version 4.3.0 "Cardinal"                                                %
 %                                                                              %
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 % ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
 %
 % Physical governing equations (EULER, NAVIER_STOKES,
-%                               WAVE_EQUATION, HEAT_EQUATION, FEM_ELASTICITY,
+%                               TNE2_EULER, TNE2_NAVIER_STOKES,
+%                               WAVE_EQUATION, HEAT_EQUATION, LINEAR_ELASTICITY,
 %                               POISSON_EQUATION)
 PHYSICAL_PROBLEM= EULER
 %
 % Specify turbulent model (NONE, SA, SA_NEG, SST)
 KIND_TURB_MODEL= NONE
 %
-% Mathematical problem (DIRECT, CONTINUOUS_ADJOINT)
+% Mathematical problem (DIRECT, ADJOINT, LINEARIZED, ONE_SHOT_ADJOINT)

```

Comment:
Thanks for the comments Heather. We'll edit the config file and commit the change.